README.md

```diff
@@ -0,0 +1,25 @@
+# Picoprobe
+
+A Picoprobe is a Raspberry Pi RP2040-based Single-Wire Debug (SWB) adaptor. You can make one from a Raspberry Pi Pico — hence the name — or almost any third-party RP2040-based board. Use it to connect your RP2040-based application board to your personal computer for debugging through `gdb`, directly or via an IDE such as Microsoft Visual Studio Code.
+
+The original version of this code only supported the Pico. This update adds support for three more boards, each of which provides a [STEMMA QT connector](https://learn.adafruit.com/introducing-adafruit-stemma-qt/what-is-stemma-qt)/[Quicc](https://www.sparkfun.com/qwiic) for easy cabling to the application board under test. These connectors are intended for use with I&sup2;C devices, but here we’re using their pins for GPIO and power.
```

Comment:
I don't think it makes sense for README.md to talk about "This update..." ? (this is the README, not a CHANGELOG)
Quicc -> Qwiic ?

Reply:
I think what is worth mentioning is the that if you are using the STEMMA/Qwiic board is that UART lines will not be passed through. Using the base board has that advantage as you don't need to plug in both your pico probe and the target board to USB.